h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -135,44 +119,22 @@ public void initMetalearnerParams(MetalearnerAlgorithm algo) {
    */
   @Override
   protected Frame predictScoreImpl(Frame fr, Frame adaptFrm, String destination_key, Job j, boolean computeMetrics, CFuncRef customMetricFunc) {
```

Comment:
@michalkurka : please review the rewrite of this `predictScoreImpl` method for Stacked Ensemble, that's also what fixes the issue PUBDEV-6266 (cf. test below).

Reply:
Looks good to me so far - I will check one more time